exercises/Neon-phenology-from-space-R.md

```diff
@@ -20,4 +20,10 @@ implicitly by sampling period for analysis.
 
 1. Plot the whole-raster mean NDVI (`cellStats()`) for Harvard Forest and SJER
    through time using different colors for the two sites. 
-2. Extract the NDVI values from all rasters for the `plot_locations` in `NEON-airborne`. Transpose this with `t()` so that there is one column for each point, make this into a dataframe, and turn the rownames into a column. Do this for both `HARV` and `SJER`. 
+2. Extract the NDVI values from all rasters for the `HARV_plots` and `SJER_plots`in `NEON-airborne/plot_locations`. Transpose these data with the following code: 
+
+	```
+	harv_ndvi_pts <- t(harv_ndvi_pts)
+	```
+
```

Comment:
I agree that this is a little awkward. How about just a little more explanation, e.g., "Extracting from a stacked raster will produce a table with one row for each point and one column for each band. Transpose this table with..."